sheet.go

```diff
@@ -110,6 +110,10 @@ func (s *Sheet) makeXLSXSheet(refTable *RefTable, styles *xlsxStyleSheet) *xlsxW
 		}
 		xRow := xlsxRow{}
 		xRow.R = r + 1
+		if row.IsCustom {
+			xRow.CustomHeight = true
+			xRow.Ht = fmt.Sprintf("%g", row.Height)
```

Comment:
It might be nicer to just set `xRow.Ht = row.Height()`, and add
```go
`func (r *Row) Height() float64 {
    if r.CustomHeight {
        return fmt.Sprintf("%g", r.Height)
    }
    return fmt.Sprintf("%g", <default here>)
}
```
It might even be better to eliminate `CustomHeight` entirely, and just check whether `r.Height` is zero here. 

It would be ideal to have a default height set, and automatically set it upon row creation with `.AddRow`, but rows are created in multiple places, so I don't know how much work it would be to ensure every place gets the default. So just checking for zero here would be simpler. Any thoughts, @tealeg?